### ISSUE TITLE
Enhancement: Release reserved

### ISSUE THREAD

**Issue by samozzy** [newtheatre/nt-tickets#110]
It's not possible to fully sell out a show when there are unclaimed reservations. Given we operate that reservations are opened up at 5-10 minutes before the start time, unclaimed reservations should not count against the `tickets remaining` number.

This could be automagic based on time or a button.

**Comment by harry-bridge** [comment]
@samozzy Yes, this should already be the case but I can't remember. I definitely implemented unclaimed reservation tracking on the db side, but whether I finished it or not I don't know
